apps/demo/config/vite/app.ts

```diff
@@ -65,6 +65,7 @@ export default defineConfig({
           join(pathRelativeToRepositoryRoot, 'node_modules'),
           join(pathRelativeToRepositoryRoot, 'node_modules/@lmc-eu/spirit-design-tokens/src/scss'),
         ],
+        silenceDeprecations: ['mixed-decls'],
```

Comment:
maybe we should add a comment why we do this 😬

Reply:
:D see what @literat said

Reply:
added